app/src/main/java/org/p2p/wallet/auth/gateway/api/response/UpdateMetadataResponse.kt

```diff
@@ -0,0 +1,8 @@
+package org.p2p.wallet.auth.gateway.api.response
+
+import com.google.gson.annotations.SerializedName
+
+data class UpdateMetadataResponse(
+    @SerializedName("status")
+    val status: Boolean,
```

Comment:
no need for field metadata from server?

Reply:
https://p2pvalidator.atlassian.net/wiki/spaces/Wallet/pages/2479030273/API+Gateway+Get+metadata#Output-parameters.1